server/src/main/java/org/opensearch/indices/cluster/IndicesClusterStateService.java

```diff
@@ -241,6 +247,7 @@ public IndicesClusterStateService(
         this.retentionLeaseSyncer = Objects.requireNonNull(retentionLeaseSyncer);
         this.sendRefreshMapping = settings.getAsBoolean("indices.cluster.send_refresh_mapping", true);
         this.remoteStorePressureService = remoteStorePressureService;
+        this.remoteStoreStatsTrackerFactory = remoteStoreStatsTrackerFactory;
```

Comment:
Where are we using this field?

Reply:
Updated. Resolving this thread. Please feel free to re-open if needed.